### PR TITLE
Add global exception handling

### PR DESCRIPTION
**Pull request by YaroslavYarmoshyk**

### CODE REVIEW

File: src/main/java/com/online/bookstore/dto/CreateBookRequestDto.java [hunk 1]
```diff
@@ -1,11 +1,18 @@
...
 
 public record CreateBookRequestDto(
+        @NotNull
         String title,
+        @NotNull
         String author,
+        @NotNull
         String isbn,
+        @NotNull
+        @Min(0)
```
Comment (MentorMA) [inline src/main/java/com/online/bookstore/dto/CreateBookRequestDto.java]: Let's validate more detail, like Not Blank, ISBN anno, limit or smth else

File: src/main/java/com/online/bookstore/dto/CreateBookRequestDto.java [hunk 2]
```diff
@@ -1,11 +1,18 @@
...
         BigDecimal price,
         String description,
```
Comment (MentorMA) [inline src/main/java/com/online/bookstore/dto/CreateBookRequestDto.java:24]: Description you can validate too

**Review by MentorMA — CHANGES_REQUESTED** [review]

File: src/main/java/com/online/bookstore/dto/CreateBookRequestDto.java (hunk 2 above)
Comment (YaroslavYarmoshyk) [reply src/main/java/com/online/bookstore/dto/CreateBookRequestDto.java:24]: added

File: src/main/java/com/online/bookstore/dto/CreateBookRequestDto.java (hunk 1 above)
Comment (YaroslavYarmoshyk) [reply src/main/java/com/online/bookstore/dto/CreateBookRequestDto.java]: validated

**Pull request merged**